src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days > 0 && days < 3) {
+    cost = 40 * days;
+  } else if (days >= 3 && days < 7) {
+    cost = 40 * days - 20;
+  } else {
+    cost = 40 * days - 50;
+  }
```

Comment:
create variables for all numbers
also, create a variable for `40 * days`, you use it three times